--- mix2005-1.1.0/mix2005/Makefile.orig	2005-04-27 08:52:15.000000000 +0200
+++ mix2005-1.1.0/mix2005/Makefile	2005-11-01 20:02:32.512632984 +0100
@@ -4,8 +4,7 @@
 include ../global_config
 
 CFLAGS += -I../guicast \
-	-I../quicktime \
-	-I$(ALSA_DIR)/include
+	-I/usr/include/quicktime
 
 PREFIX=/usr
 
@@ -28,11 +27,11 @@
 THEME_DATA := $(OBJDIR)/theme_data.o
 
 ifeq ($(OBJDIR), x86_64)
-LIBS := -L/usr/X11R6/lib64
+LXLIBDIR := -L/usr/X11R6/lib64
 else
-LIBS := -L/usr/X11R6/lib
+LXLIBDIR := -L/usr/X11R6/lib
 endif
-
+LIBS := $(LXLIBDIR)
 
 
 LIBS += -lGL \
@@ -44,7 +43,7 @@
 	-lz  \
 	-ldl  \
 	$(XXF86VM) \
-	$(ALSA_DIR)/$(OBJDIR)/libasound.a \
+	-lasound \
 	$(THEME_DATA)
 
 ifeq ($(HAVE_XFT), y)
@@ -59,7 +58,7 @@
 	$(CC) -o $(OUTPUT) \
 		$(OBJS) \
 		../guicast/$(OBJDIR)/libguicast.a \
-		../quicktime/$(OBJDIR)/libquicktime.a \
+		-lquicktime \
 		$(LIBS)
 
 $(OBJS):
--- mix2000-1.0.1/global_config.orig	2004-04-05 03:26:48.000000000 +0200
+++ mix2000-1.0.1/global_config	2004-06-04 22:20:58.341931584 +0200
@@ -71,10 +71,6 @@
 CFLAGS += $(ALSA_CFLAGS)
 endif
 
-ifeq ($(OBJDIR), alpha)
-CXXREPOSITORY = cxx_repository/*.o
-endif
-
 $(shell sh -c 'if ! test -d $(OBJDIR)\; then mkdir $(OBJDIR)\; fi' )
 
 ########################## LINKER
--- mix2005-1.1.0/guicast/Makefile.orig	2005-07-19 02:02:23.000000000 +0200
+++ mix2005-1.1.0/guicast/Makefile	2005-11-01 20:22:56.719525248 +0100
@@ -2,9 +2,8 @@
 
 # Need the freetype include that matches the X server for Xft rendering
 #FREETYPE_DIR := ..//usr/X11R6/include/freetype2
-FREETYPE_DIR := $(shell expr ../freetype*/include )
 
-CFLAGS += -I../quicktime -I$(FREETYPE_DIR)
+CFLAGS += -I/usr/include/quicktime -I/usr/include/freetype2
 
 # must only define here so applications using libpng don't see any of the XFT
 # includes at all.  They use a conflicting setjmp.
@@ -25,6 +24,17 @@
 LFLAGS += -L/usr/X11R6/lib64
 endif
 
+ifeq ($(OBJDIR), alpha)
+BOOTSTRAPFLAGS := -DBOOTSTRAP="\"objcopy -B alpha -I binary -O elf64-alpha\""
+endif
+
+ifeq ($(OBJDIR), ppc)
+BOOTSTRAPFLAGS := -DBOOTSTRAP="\"objcopy -B powerpc -I binary -O elf32-powerpc\""
+endif
+
+ifeq ($(OBJDIR), sparc)
+BOOTSTRAPFLAGS := -DBOOTSTRAP="\"objcopy -B sparc -I binary -O elf32-sparc\""
+endif
 
 $(shell sh -c 'if ! test -d $(OBJDIR)\; then mkdir $(OBJDIR)\; fi' )
 
